### ISSUE TITLE
Using arrow keys for menu selection causes error

### ISSUE THREAD

**Issue by brantb** [profclems/glab#821]
**Description**
When making a selection from a menu, pressing any arrow key causes an error and the app quits.

Other keys (text, enter) seem to work as expected, with the exception of Home and End (which cause the same error)

```
bash-3.2$ glab auth login
? What GitLab instance do you want to log into?  [Use arrows to move, type to filter]
> GitLab.com
  GitLab Self-hosted Instance
[press down arrow key]
could not prompt: Unexpected Escape Sequence: ['\x1b' 'O']
bash-3.2$
```

**Your Environment**

* Version used: glab version 1.20.0 (2021-08-11), installed via `brew`
* Operating System and version: macOS 11.5.1

**Comment by zemzale** [comment]
Thanks for reporting the issue @brantb 

Sadly I can't reproduce the error either on macOS (tried both zsh, and bash) neither Linux (tried bash and zsh).

This seems to be an issue upstream https://github.com/AlecAivazis/survey/issues/341. Terminal escape codes are bit of magic to me that I don't full grasp, so maybe someone else can help with the workaround.

**Comment by brantb** [comment]
Yup, that looks like the exact same issue. I'm using PowerShell as well but tested in bash by launching `bash` **from** a PowerShell prompt, which is why you couldn't reproduce it. When I launch bash/zsh directly (from a separate iTerm2 profile) I don't hit the error either.

**Comment by brantb** [comment]
I'll close this because it's covered pretty well by the issue you linked (and is fixed in the latest PowerShell 7.2 preview anyway), thanks for looking into this!
